tests/perf/perf_tests.cc

```diff
@@ -41,6 +41,7 @@
 #include <seastar/core/reactor.hh>
 
 #include <signal.h>
+#include <type_traits>
```

Comment:
could you please move this `#include` up, so it is closer to the other standard library headers. would be better to group similar headers together.